pkg/task/restore.go

```diff
@@ -25,12 +25,14 @@ import (
 )
 
 const (
-	flagOnline   = "online"
-	flagNoSchema = "no-schema"
+	flagOnline             = "online"
+	flagNoSchema           = "no-schema"
+	flagSwitchModeInterval = "switch-mode-interval"
 
 	defaultRestoreConcurrency = 128
 	maxRestoreBatchSizeLimit  = 256
 	defaultDDLConcurrency     = 16
+	defaultSwitchInterval     = 5 * time.Minute
```

Comment:
We should add this flag to raw restore too or give the `rc.switchModeInterval` field a default value, otherwise we will call `time.NewTicker(0)` and make a panic...

Reply:
yes, I moved this `switchModeInterval` to common config